libnavigation-ui/src/main/java/com/mapbox/navigation/ui/feedback/FeedbackBottomSheet.java

```diff
@@ -159,9 +160,28 @@ public void onSaveInstanceState(@NonNull Bundle outState) {
     }
   }
 
+  @Override
+  public void onStop() {
+    super.onStop();
+    if (countdownAnimation.isRunning()) {
+      dismissCommand = delayedDismissCommand;
+    }
+  }
+
+  @Override
+  public void onResume() {
+    super.onResume();
+    if (dismissCommand != null) {
+      dismissCommand.invoke();
+    }
+    dismissCommand = null;
+  }
+
   @Override
   public void onAnimationEnd(Animator animation) {
-    FeedbackBottomSheet.this.dismiss();
+    if (FeedbackBottomSheet.this.isResumed()) {
+      FeedbackBottomSheet.this.dismiss();
+    }
```

Comment:
should we set the `dismissCommand` in the `else` condition?
The current way to set it at onStop will have another issue that: if the user switches back to foreground before the countdown animation ends, the FeedbackBottomSheet will be dismissed too.

Reply:
yes, I don't know what I was thinking.